scripts/kc_data_transfer/src/KeyCloakTransfer.py

```diff
@@ -328,7 +326,7 @@ def get_roles(self, app_id) -> List[FamRole]:
         data = resp.json()
         return data
 
-    def get_role(
+    def get_role_id(
```

Comment:
this method previously called get_role,  but is not in use at all, changed it to get_role_id and let it returns the role_id